packages/adapter-netlify/README.md

```diff
@@ -36,6 +36,27 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 
 It's recommended that you add the `build` and `functions` folders (or whichever other folders you specify) to your `.gitignore`.
 
-## Extra Resources
-
-Guide for SvelteKit + Netlify Forms users: https://dev.to/swyx/how-to-use-sveltekit-with-netlify-forms-5gmj
+## Troubleshooting
+
+### How to make Netlify forms work?
+
+1. Add the following [prerendering configuration](https://kit.svelte.dev/docs#configuration-prerender) to `svelte.config.js`:
+   ```js
+   prerender: {
+       crawl: true,
+       enabled: true,
+       force: true,
+       pages: ['*']
+   }
+   ```
+2. Create your Netlify HTML form as described [here](https://docs.netlify.com/forms/setup/#html-forms), e.g. as `/routes/contact.svelte`. (Don't forget to add the hidden "form-name" input element!)
+3. Tell SvelteKit to prerender your `contact.svelte` page component by adding:
```

Comment:
If we're specifying `force: true` you shouldn't need to do this. I'm not sure why either is necessary though